src/monster.cpp

```diff
@@ -845,6 +854,11 @@ bool monster::made_of( phase_id p ) const
     return type->phase == p;
 }
 
+void monster::set_goal( tripoint &p )
```

Comment:
Why does this even require a non-const reference?
```suggestion
void monster::set_goal( const tripoint &p )
```